lib/active_record_shards/default_slave_patches.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 module ActiveRecordShards
   module DefaultSlavePatches
-    def self.wrap_method_in_on_slave(class_method, base, method)
+    def self.wrap_method_in_on_slave(class_method, base, method, force_on_slave = false)
```

Comment:
explicit keyword arg would be nice
```suggestion
    def self.wrap_method_in_on_slave(class_method, base, method, force:)
```

... and if client code uses it then

```suggestion
    def self.wrap_method_in_on_slave(class_method, base, method, force: false)
```

Reply:
I prefer keyword args, but didn't want to mix kw and non-kw, and didn't want to change the signature.